synedrion/Cargo.toml

```diff
@@ -62,7 +62,13 @@ private_benches = ["criterion"]
 bench = true
 name = "bench"
 harness = false
+path = "benches/bench.rs"
 
+[[bench]]
+bench = true
+name = "pow"
+harness = false
+path = "benches/pow.rs"
```

Comment:
@fjarri I left these in, but they are not particularly useful long-term I think. I wrote them to check on progress as I was changing code, but I think we should remove them now.